### PR TITLE
Use === instead of ==

### PR DESCRIPTION
**Pull request by bhaskarmelkani**
Updated directory structure.

### CODE REVIEW

**Comment by bhaskarmelkani** [comment]
Original PR #152

File: _posts/en/2016-xx-xx-use_===_instead_of_==.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+
+title: Use === instead of ==.
+tip-number: xx
```
Comment (loverajoel) [inline _posts/en/2016-xx-xx-use_===_instead_of_==.md]: update the tip number to 24

**Comment by loverajoel** [comment]
update the fila name with today date

File: _posts/en/2016-xx-xx-use_===_instead_of_==.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+tip-username: bhaskarmelkani
+tip-username-profile: https://www.twitter.com/bhaskarmelkani
+tip-tldr: === comparision in safe and faster.
+
+categories:
+    - en
+---
+
+The `==` (or `!=`) operator performs an automatic type conversion if needed. The `===` (or `!==`) operator will not perform any conversion. It compares the value and the type, which could be considered faster ([jsPref](http://jsperf.com/strictcompare)) than `==`.
```
Comment (loverajoel) [inline _posts/en/2016-xx-xx-use_===_instead_of_==.md]: I think that all of this phrase can be the `tip-tldr`

**Comment by loverajoel** [comment]
if you can, please squash your commits

**Comment by bhaskarmelkani** [comment]
Okay

**Comment by bhaskarmelkani** [comment]
Since this was forked from master, so squashing was creating conflicts and history was breaking.
Created ready to merge PR #155

**Pull request closed without merging**